### DISCUSSION TITLE
Confusion on examples and Npm latest version

### DISCUSSION THREAD

**Discussion by AlexisIoannou** [clauderic/dnd-kit#451] [General]
Hiya, first of all amazing job on this package :), it looks very polished and flexible for various use cases compared to the current alternatives and very promising. 

Just wanted to mention something that might cause some confusion for some people trying out the examples. Me eagerly trying the tree example to get a grasp of how it works, I installed the latest npm, and then copied over the tree example on my local to test it out but I was getting compile errors. The issue was that the latest npm I installed( @dnd-kit.core@3.1.1) was not the same as the examples, which were on master. Got a working tree example by switching to @dnd-kit.core@3.1.1 branch and getting the compatible code from there. My suggestion would be to maybe have something in the docs to point that out, or have the examples match the latest version maybe.
 
Loving the npm so far :)!

**Comment by clauderic** [comment]
Hey @AlexisIoannou, good shout, I will add something to the docs about the release strategy in the near future.

To use the code that is on the master branch, install the npm packages tagged with `@next` on npm.

Refer to https://github.com/clauderic/dnd-kit/pull/342#issuecomment-871842286 for additional details
